packages/third-parties/src/google/ga.tsx

```diff
@@ -54,14 +54,14 @@ export function GoogleAnalytics(props: GAParams) {
   )
 }
 
-export function sendGAEvent() {
+export const sendGAEvent = (...args: Object[]) => {
   if (currDataLayerName === undefined) {
     console.warn(`@next/third-parties: GA has not been initialized`)
     return
   }
 
   if (window[currDataLayerName]) {
-    window[currDataLayerName].push(arguments)
+    window[currDataLayerName].push(args)
```

Comment:
We might want to leave this one as `arguements`. It seems like this does not work with Arrays, but works with [arguments object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments). Additionally, when using `window.gtag(...)` an `Arguments()` is pushed instead of `Array()`.

Might be related, see: https://stackoverflow.com/questions/60400130/google-analytics-datalayer-push-not-working-with-array-of-variables#60410151

Reply:
Not sure if TS will like it:
```suggestion
    window[currDataLayerName].push(arguements)
```

Reply:
Yes, this should not have been changed from `arguments`...